package.json

```diff
@@ -31,17 +30,15 @@
     "gulp-sass": "^2.3.2",
     "gulp-sourcemaps": "^1.6.0",
     "gulp-util": "^3.0.7",
-    "hogan.js": "3.0.2",
     "marked": "^0.3.6",
-    "minimist": "0.0.8",
```

Comment:
This isn't being used to set the port anymore, but [looks like it might be useful](https://www.npmjs.com/package/minimist), if we can keep it in - in case others are using it.